SDL_ttf.c

```diff
@@ -3035,6 +3035,16 @@ void TTF_SetFontKerning(TTF_Font *font, int allowed)
 #endif
 }
 
+void TTF_GetFontBBox(const TTF_Font *font, TTF_BBox *bbox)
+{
+    /* Recalculate FT_Face's bbox from font units to pixels */
```

Comment:
You should make sure the font is valid, see the other API functions for examples.

Reply:
I guess I have to test that `font` pointer is valid? By the way, I noticed that not all functions have this check, only some do.